dask/array/core.py

```diff
@@ -607,6 +609,8 @@ def __neg__(self):
         return elemwise(operator.neg, self)
     def __or__(self, other):
         return elemwise(operator.or_, self, other)
+    def __pos__(self):
+        return self
```

Comment:
arguably, this should actually call unary plus on everything, though numpy seems to also treat it as a no-op:
```
In [4]: +np.array(['asdf'])
Out[4]:
array(['asdf'],
      dtype='|S4')
```